src/.nuget/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00040" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-11580" />
   <package id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+  <package id="dnx-mono" version="1.0.0-beta5-11596" />
```

Comment:
I'll split it in a separate change after I get the other build issues worked out. To be agnostic I'll split out the Windows only package as well.